plugfest-2020/README.md

```diff
@@ -54,11 +54,12 @@ See [this example](./vendors/transmute/index.js).
 2. The Issuer's Issue Credential HTTP API MUST require `"credential"` in the body of the POST request. The field `"credential"` MUST be conformant to [Verifiable Credentials Data Model 1.0](https://www.w3.org/TR/vc-data-model/).
 3. The Issuer's Issue Credential HTTP API MUST support the issuance of credentials with at least 2 different DID methods as the `"issuer"` on a Verifiable Credential.
 4. The Issuer's Issue Credential HTTP API MUST return a 400 HTTP response status code when the request is rejected.
-5. The Issuer's Issue Credential HTTP API MUST reject if the value of `"credential.issuer"` or `"credential.issuer.id"` in the body of the POST request is not a DID.
+5. The Issuer's Issue Credential HTTP API MUST return a Verifiable Credential with the value of its `"issuer"` or `"issuer.id"` as a URI in the body of the response.
```

Comment:
What if the issuer wants to validate that the issuer DID is related to the linked data proof assertion?

Reply:
@troyronda I don't understand the question, can you pull it into an issue to continue?

Reply:
I created this issue as the more general consideration: https://github.com/w3c-ccg/vc-examples/issues/67